README.md

```diff
@@ -16,4 +16,7 @@ Then create a file named `.eslintrc.json` with following contents in the root fo
 }
 ```
 
+**NB:** if you're using [the extension ESLint for VS Code](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint),
```

Comment:
I don't like this, it's too specific for a one use case. In exchange, I propose to change the readme mentioning the default config file should be just `.eslintrc`. In fact, it's the file created when `eslint --init`.

Reply:
Works for me